examples/json_map/README.md

```diff
@@ -0,0 +1,25 @@
+# JSON Map Reader
+
+## What is this reader?
+
+This reader is designed to allow users of pynxtools to convert their existing data with the help of a map file. The map file tells the reader what to pick from your data files and convert them to FAIR NeXus files. The following formats are supported as input files:
+* HDF5 (any extension works i.e. h5, hdf5, nxs, etc)
+* JSON
+* Python Dict Objects Pickled with [pickle](https://docs.python.org/3/library/pickle.html). These can contain [xarray.DataArray](https://docs.xarray.dev/en/stable/generated/xarray.DataArray.html) objects as well as regular Python types and Numpy types.
+
+## How to run these examples?
+
+### Automatically merge partial NeXus files
+```console
+user@box:~$ dataconverter --nxdl NXiv_temp --input-file voltage_and_temperature.nxs --input-file current.nxs --merge-partial --output auto_merged.nxs
```

Comment:
Could this not be the default route when no reader is specified?

Reply:
We can have it as the default. I just didn't wanna take big steps like that without discussion. There isn't any other generic reader that could benefit from a default, right? So I can set this as the default.

Reply:
Yeah, now I'm not fully sure about it. Could be that users might just have forgotten to specify the reader, so if we use it as default we might show an additional info text that the partial merging was used (even though the reader could figure it out from the source file it got: nxs files -> merging; any other file -> probably user forgot to specify the reader. but I don't know how cumbersome it is to implement this)

Reply:
It's done. Commit coming up in a minute.